### ISSUE TITLE
Requests

### ISSUE THREAD

**Issue by dherl0623** [jclarke0000/MMM-MyCommute#9]
I wondered if requests are still made for destinations that are hidden during the certain times that are set? For instance, if I had a destination hidden from 11PM to 8AM is it getting travel data every 10 minutes during the night? I was just wondering, and thought that if it is requesting that data that it would kinda be a waste of your request amount.

Thanks for an awesome module by the way! I love the functionality that you have provided with this.

**Comment by jclarke0000** [comment]
No, requests are only made for destinations that are currently in window. Part of the reason this ability exists is to cut down on the number of requests made to Google's API.
